recipes/aws-crt-cpp/all/conanfile.py

```diff
@@ -46,22 +47,23 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("aws-c-common/0.8.2")
+        self.requires("aws-c-cal/0.5.13", transitive_headers=True)
```

Comment:
Out of curiosity — can these transitive_headers additions be dropped after all libraries from https://github.com/conan-io/conan-center-index/pull/17125 are aligned?

Reply:
No, these headers are really transitive.